activesupport/test/notifications/event_assertions_test.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require_relative "../abstract_unit"
+require "active_support/notifications/event_assertions"
+
+module ActiveSupport
+  module Notifications
+    class EventAssertionsTest < ActiveSupport::TestCase
+      include EventAssertions
+
+      def test_assert_event
+        assert_event("post.submitted", title: "Cool Post") do
+          ActiveSupport::Notifications.instrument("post.submitted", title: "Cool Post")
+        end
+
+        assert_event("post.submitted") do # payload omitted
+          ActiveSupport::Notifications.instrument("post.submitted", title: "Cool Post")
+        end
+
+        error = assert_raises(Minitest::Assertion) {
+          assert_event("post.submitted", title: "Cool Post") { nil } # no events
+        }
+        assert_match(/No post.submitted events were found/, error.message)
```

Comment:
Could the assertions in this style utilize the [assert_raises](https://edgeapi.rubyonrails.org/classes/ActiveSupport/Testing/Assertions.html#method-i-assert_raises) `:match` option?

```suggestion
        assert_raises Minitest::Assertion, match: /No post.submitted events were found/ do
          assert_event("post.submitted", title: "Cool Post") { nil } # no events
        end
```

Reply:
Great suggestion. Wasn't aware of that `assert_raises` parameter. Nice! Will push a cleanup.